addons/website_sale/static/src/js/tours/website_sale_shop.js

```diff
@@ -44,6 +44,11 @@ odoo.define("website_sale.tour_shop", function (require) {
         run: function (actions) {
             actions.auto(".modal-footer .btn-secondary");
         },
+    }, {
+        trigger: "button.o_we_add_snippet_btn",
+        content: _t("Click here to add another block."),
```

Comment:
Remove the content key, it will never be displayed and this adds a term to translate for nothing ;)

Reply:
and position too is useless ;)